src/app/auto-correct/auto-correct.component.ts

```diff
@@ -12,13 +12,24 @@ import * as fromRoot from '../reducers';
 
 export class AutoCorrectComponent implements OnInit {
   query$: any;
+  wholequery$: any;
   suggestion: any;
   sugflag: boolean;
+  isPage1: boolean;
```

Comment:
why do we need another variable here. isn't it enough to check `presentPage===1`